### PR TITLE
Internationalization

### PR DESCRIPTION
**Pull request by phil-monroe**
Working on #30 

#### Circles
- [x] cirlces/index
- [x] cirlces/new
- [x] circles/show 
- [x] circles/edit
- [x] circles/_form

#### Other
- [x] layouts/shared/_header
- [x] sessions/new

### CODE REVIEW

**Comment by dottorer** [comment]
@phil-monroe  Thanks for setting up I18N. I presume I can clone the repo locally and find the respective config/locales/*/*/de.yml files (e.g. config/locales/views/circles/de.yml) for me to translate. That would work well.

Also, would we have a URL parm initially (like "?lang=de") to switch context. Until we have a user-level profile setting to force it...

**Comment by tmandke** [comment]
Added internationalization setup for auth. Was also trying to use https://github.com/amatsuda/i18n_generators to auto transalate all the models/attributes. Google seems to detect that its a bot and ask a capsa. So I guess we can use it to create a file with all the proper keys but maintain the values manually. (unless u can get it to work)

**Comment by tmandke** [comment]
Also, I think you forgot about the flash messages in the controllers.

**Comment by dottorer** [comment]
I think having resource files ready and then translate them manually is the way to go. I can organize this for many languages here in Europe with friends in the software translation business.

**Comment by phil-monroe** [comment]
@tmandke, awesome! and yeah, flash messages...

**Comment by phil-monroe** [comment]
@tmandke: added translations for flash messages. I'd like to merge this is ASAP so other people can start translating. Any other problems you see? I plan on adding translations for the working groups and tasks CRUD pages once we figure out a more concrete UX for them.

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def edit
   def create
     respond_to do |format|
       if @task.save
-        format.html { redirect_to @circle, notice: 'Task was successfully created.' }
+        format.html { redirect_to @circle, notice: t('flash.created', name: 'Task') }
```
Comment (tmandke) [inline app/controllers/tasks_controller.rb]: Task is not translated here and in the following flashes as well

File: app/controllers/working_groups_controller.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@ def edit
   def create
     respond_to do |format|
       if @working_group.save
-        format.html { redirect_to [@circle], notice: 'Working group was successfully created.' }
+        format.html { redirect_to [@circle], notice: t('flash.created', name: 'Working group') }
```
Comment (tmandke) [inline app/controllers/working_groups_controller.rb]: Working Group is not translated here and in the following flashes as well

**Comment by tmandke** [comment]
Looks good once the above 2 things are fixed.

**Comment by phil-monroe** [comment]
:sweat: trying to be too smart for my own good...

**Comment by phil-monroe** [comment]
Fixed!

**Pull request merged**